modules/codemod/lib/v6/spec/deprecateHeader.spec.ts

```diff
@@ -8,7 +8,42 @@ describe('Canvas Kit Deprecate Header Codemod', () => {
   context('when transforming header imports', () => {
     it('should ignore non-canvas-kit imports', () => {
       const input = `import Header, {GlobalHeader} from "@workday/some-other-library";`;
-      const expected = `import Header, {GlobalHeader} from "@workday/some-other-library";`;
+      const expected = `${input}`;
```

Comment:
I started doing this for test cases where nothing changed. I hope it makes it more obvious that no change is expected. Otherwise you might scan the `input` and `expected` and not tell right away if there is a difference.